backend/src/helpers.py

```diff
@@ -14,3 +16,12 @@ def download_csv(url, fp):
 def log(message):
     if print_logs:
         print(message)
+
+def maximise_field_size_limit():
+    maxInt = maxsize
```

Comment:
I'd rather have
```python
def maximise_field_size_limit(max_int=maxsize):
```
for the sake of not using nonlocal variables. I also changed the variable name to adhere to the recommended Python style (and the style of this codebase).

Reply:
Can we fix this @Andrea-Valentini?

Reply:
finally It should be fine with the last commit.